js/runner.go

```diff
@@ -621,11 +622,26 @@ func (u *VU) runFn(
 	u.Runtime.Set("__ITER", u.Iteration)
 	u.Iteration++
 
+	defer func() {
+		if r := recover(); r != nil {
+			gojaStack := u.Runtime.CaptureCallStack(20, nil)
+			err = fmt.Errorf("a panic occurred in VU code but was caught, %s", r)
```

Comment:
Nit: the comma feels out of place here. A colon would be more intuitive here when reading the error message as it's referencing the original one.

Reply:
fixed in d1db643